app/error_messages.py

```diff
@@ -2,7 +2,9 @@
 INVALID_WHITESPACE_FOUND = "Found invalid white space(s) in schema text"
 DUPLICATE_ID_FOUND = "Duplicate id found"
 FOR_LIST_NEVER_POPULATED = "for_list is not populated by any ListCollector blocks"
-MULTIPLE_FOR_LISTS = "Multiple for_lists in a section when section summary enabled"
+MULTIPLE_LIST_COLLECTORS_WITH_LIST_SUMMARY_ENABLED = (
+    "Multiple list collectors for same list in a section when list summary enabled"
+)
```

Comment:
I don't think this message is very clear. Maybe `Section cannot have multiple list collectors that populate the same list when summary with items are enabled.`

I think we need to do a piece to update all validator error messages to ensure they describe what has happened and how to fix it.